### PR TITLE
Add bravery menu to the hamburger menu

### PR DESCRIPTION
**Pull request by diracdeltas**
Auditors: @bbondy

### CODE REVIEW

File: js/components/main.js [hunk 1]
```diff
@@ -102,6 +102,19 @@ class Main extends ImmutableComponent {
...
+  onHamburgerMenu () {
+    let settings = {}
+    const resourceNames = ['httpsEverywhere', 'adblock', 'trackingProtection', 'adInsertion']
+    resourceNames.forEach((name) => {
+      let enabled = this.props.appState.getIn([name, 'enabled'])
+      settings[name] = enabled === undefined ? AppConfig[name].enabled : enabled
+    })
+    // whether the current page is bookmarked. needed to re-initialize the
+    // application menu.
+    settings.bookmarked = this.refs.navBar.bookmarked
```
Comment (diracdeltas) [inline js/components/main.js]: I guess we're no longer using refs? lmk if i should update this to be consistent with https://github.com/brave/browser-laptop/pull/316/files

Comment (bbondy) [inline js/components/main.js]: yes the approach in https://github.com/brave/browser-laptop/pull/316/files is better thanks.

File: js/components/main.js [hunk 2]
```diff
@@ -102,6 +102,19 @@ class Main extends ImmutableComponent {
     // TODO
   }
 
+  onHamburgerMenu () {
+    let settings = {}
+    const resourceNames = ['httpsEverywhere', 'adblock', 'trackingProtection', 'adInsertion']
```
Comment (bbondy) [inline js/components/main.js]: would be better if we pulled these in from a constants file in case the names ever change.

**Comment by bbondy** [comment]
Couple of minor things, please feel free to merge with edits.

**Pull request merged**